projects/power_distribution/src/main.c

```diff
@@ -29,6 +29,7 @@
 #include "publish_data.h"
 #include "publish_data_config.h"
 #include "rear_strobe_blinker.h"
+#include "smoke_uv_cutoff.h"
```

Comment:
You don't actually need to #include this since main.c doesn't use any of the symbols (functions, etc) it declares.